OA_Email.Tests/OA_Email.Tests.csproj

```diff
@@ -9,7 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="AutoFixture" Version="4.18.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
+    <PackageReference Include="Moq" Version="4.20.69" />
```

Comment:
Precisamos trocar o Moq
ref.: https://dev.to/pbouillon/the-moq-gate-you-either-die-a-hero-8